doc/source/gettingstarted/migration.rst

```diff
@@ -32,3 +32,6 @@ Overview of API Changes
     * - Create a custom neighborlist from arrays.
       - ``freud.locality.NeighborList.from_arrays(..., distances=...)``
       - ``freud.locality.NeighborList.from_arrays(..., vectors=...)``
+    * - Compare environments of all particle pairs with ``EnvironmentCluster``.
+      - ``env_cluster.compute(..., global_search=True)``
+      - ``env_cluster.compute(..., neighbors=all_pairs_nlist, ...)``
```

Comment:
Maybe worth showing how to construct `all_pairs_nlist`. Depends on #1051's new feature.

Reply:
Good suggestion. However, I'd like to wait until we have an official release out with the changes from #1051 on it before adding the utility method in the example.